src/index.js

```diff
@@ -1,10 +1,25 @@
 /* eslint-disable no-param-reassign */
-export default function(breakpoints) {
+export default function(breakpoints, overlapp) {
```

Comment:
Let's make this an options `object`. I'm sure there are going to be more in the future.

Reply:
Clarification.